src/cpu/flexcpu/simple_dataflow_thread.cc

```diff
@@ -754,9 +752,16 @@ SDCPUThread::onExecutionCompleted(weak_ptr<InflightInst> inst, Fault fault)
                     squashUpTo(inst_ptr, true);
 
                     // Notify branch predictor of incorrect prediction
-
-                    _cpuPtr->getBranchPredictor()->squash(inst_ptr->seqNum(),
```

Comment:
I don't think this was the right squash function to call -- probably should just call the other squash function right now.

Reply:
Note that if you add a commit callback, the callback will never be called, since the faulting instruction is not committed.